recipes_source/compiling_optimizer_lr_scheduler.rst

```diff
@@ -0,0 +1,125 @@
+(beta) Running the compiled optimizer with an LR Scheduler
+==========================================================================================
+
+**Author:** `Michael Lazos <https://github.com/mlazos>`_
+
+The optimizer is a key algorithm for training any deep learning model.
+In this example, we will show how to pair the ``torch.compile``d optimizer
```

Comment:
Can we not say ``torch.compile``d since it's not really a verb. Maybe something like:
```suggestion
In this example, we will show how to pair the an optimizer, which has been compiled using ``torch.compile``,
```